io.github.purplehorrorrus.Meridius.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
This is something you should ideally maintain and distribute from your own repo/with the tarball.